README.mkd

```diff
@@ -412,6 +412,15 @@ Control the code generation for scalar memory access with strict alignment (also
 
 NOTE: The compiler may generate misaligned access if the program violates the alignment assumption.
 
+
+### `-mscalar-strict-align`/`-mno-scalar-strict-align`
+
+Aliases for -mstrict-align/-mno-strict-align.
```

Comment:
Is this correct? I.e., does `-mstrict-align` not cover vector memory accesses?
If so, then we should add this detail to #49.
If not, then we need to state here, that it only covers scalar memory accesses.

Reply:
I believe #49 does say "scalar memory accesses"

Reply:
Oh, right, it even uses these words multiple times.
There must have been a rendering issue in my browser or so... ;)